### ISSUE TITLE
/datapack enable stops the cut

### ISSUE THREAD

**Issue by meishiffx** [thraaawn/BonsaiTrees#334]
Can't auto trees after /datapack enable on server.
Mods version: 3.1.0.

**Comment by meishiffx** [comment]
Oh, my brother, you haven’t updated the version yet.（

**Comment by Sabbelchen** [comment]
I can confirm this problem. Happend to me in Version 1.18.2. and bonsaitrees3-3.1.0.jar

How can I solve this problem?

**Comment by davenonymous** [comment]
If you have the time to re-test this issue with the [latest version](https://www.curseforge.com/minecraft/mc-mods/bonsai-trees/files/5632262), it would be greatly appreciated.

**Comment by Sabbelchen** [comment]
I have tested it on my local server because I can't test it on the server where I'm playing.
First test was with the old version of the bonsai trees mod and the same problem happend  -> Server crashes and the bonsai tress stoped working.

After updating the mod I've tried it again and the server runs normal. No crash and the bonsai pots are working fine after using /datapack.

Thanks for updating the mod! ♥
